internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -180,9 +183,38 @@ func (ms *metadataStorage) getTableName(ctx context.Context) (string, error) {
 	return metadata.table, nil
 }
 
+// renameCollection renames metadataStorage.collection.
+func (ms *metadataStorage) renameCollection(ctx context.Context, to string) error {
+	metadata, err := ms.get(ctx, true)
+	if err != nil {
+		return lazyerrors.Error(err)
+	}
+
+	if metadata.table == "" || metadata.collection == "" {
```

Comment:
if metadata is returned, these fields are never empty. there shouldn't be a field `collection` in metadata, the collection is stored in the `_id` field of metadata.